src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.figureType = "circle";
+        this.radius = radius;
+        this.area = Math.PI * radius * radius;
```

Comment:
Please, follow the task. Create interface for area calculation and separate method. Also follow [this](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-put-all-behaviour-into-a-single-interface-if-the-methods-are-conceptually-different-from-each-other).